webui/react/src/pages/TrialDetails/TrialChart.tsx

```diff
@@ -105,7 +105,7 @@ const TrialChart: React.FC<Props> = ({
   return (
     <Section bodyBorder options={options} title="Metrics">
       <div className={css.base}>
-        {chartData[0].length === 0 ?
+        {chartData[0].length < 2 ? // this prevents an issue where the chart doesnt follow
```

Comment:
this is intentional.

Reply:
**question:** not following... 🤪  is this to just ensure the chart has more than one data point before plotting? We we have some single trial experiments where search validation metric is only generated once, in those cases wouldn't we get `No data to plot` message?

Reply:
That is just for toy/testing examples, no? It wouldn't make sense to have validation metric generated only once. 

And in any case, it doesn't make sense/looks silly to graph just one data point.